cmake/EinsumsManageFFTLibraries.cmake

```diff
@@ -1,9 +1,9 @@
 include_guard()
 
 # If MKL is found then we must use it.
-if (NOT EINSUMS_FFT_LIBRARY MATCHES off AND EINSUMS_LINALG_VENDOR STREQUAL MKL)
+if (NOT EINSUMS_FFT_LIBRARY MATCHES off AND EINSUMS_LINALG_VENDOR STREQUAL "MKL")
```

Comment:
Was there a reason MKL needs to be in quotes? There are examples in other files where this isn't the case.